portability-core/src/main/java/org/dataportabilityproject/serviceProviders/rememberTheMilk/RememberTheMilkServiceProvider.java

```diff
@@ -34,16 +36,18 @@
  */
 final class RememberTheMilkServiceProvider implements ServiceProvider {
     private final AppCredentials appCredentials;
-    private final RememberTheMilkAuth rememberTheMilkAuth;
+    private final static ImmutableList<PortableDataType> SUPPORTED_DATA_TYPES = ImmutableList
+        .of(PortableDataType.TASKS);
+    private final RememberTheMilkAuth importAuth;
+    private final RememberTheMilkAuth exportAuth;
 
     @Inject
     RememberTheMilkServiceProvider(AppCredentialFactory appCredentialFactory) throws IOException {
         this.appCredentials = appCredentialFactory.lookupAndCreate("RTM_KEY", "RTM_SECRET");
-        this.rememberTheMilkAuth = new RememberTheMilkAuth(
-            new RememberTheMilkSignatureGenerator(
-            appCredentials,
-            null
-        ));
+        this.importAuth = new RememberTheMilkAuth(
```

Comment:
I'm not sure what our final instantiation model will be but I rather lazy init these rather than construct in the constructor in case this service is not used as part of a job.

Reply:
done.